.circleci/config.yml

```diff
@@ -137,15 +137,15 @@ commands:
       - run:
           name: Install Kubectl
           command: |
-            [[ -f /usr/local/bin/kubectl ]] && exit 0
+            [[ -f /home/circleci/bin/kubectl ]] && exit 0
             cd /tmp
             curl -LO https://storage.googleapis.com/kubernetes-release/release/<< parameters.version >>/bin/linux/amd64/kubectl
             chmod +x ./kubectl
-            sudo mv ./kubectl /usr/local/bin/kubectl
+            sudo mv ./kubectl /home/circleci/bin/kubectl
       - save_cache:
           key: "v1-kubectl-binary-<< parameters.version >>"
           paths:
-            - /usr/local/bin/kubectl
+            - /home/circleci/bin/kubectl
```

Comment:
Existing cache from prior runs will contain the old path, you have to bust it somehow.

Reply:
> Existing cache from prior runs will contain the old path, you have to bust it somehow.

Updated kubectl cache to v2.